utils/Makefile

```diff
@@ -62,45 +63,45 @@ viewcover:
 # Conditional virtualenv building strategy taken from this great post
 # by Marcel Hellkamp:
 # http://blog.bottlepy.org/2012/07/16/virtualenv-and-makefiles.html
-venv: oo-installenv/bin/activate
-oo-installenv/bin/activate: test-requirements.txt
+$(VENV): $(VENV)/bin/activate
+$(VENV)/bin/activate: test-requirements.txt
 	@echo "#############################################"
 	@echo "# Creating a virtualenv"
 	@echo "#############################################"
-	test -d venv || virtualenv $(NAME)env
-	. $(NAME)env/bin/activate && pip install setuptools==17.1.1
-	. $(NAME)env/bin/activate && pip install -r test-requirements.txt
-	touch $(NAME)env/bin/activate
+	test -d $(VENV) || virtualenv $(VENV)
+	. $(VENV)/bin/activate && pip install setuptools==17.1.1
+	. $(VENV)/bin/activate && pip install -r test-requirements.txt
+	touch $(VENV)/bin/activate
 #       If there are any special things to install do it here
-#       . $(NAME)env/bin/activate && INSTALL STUFF
+#       . $(VENV)/bin/activate && INSTALL STUFF
 
-ci-unittests:
+ci-unittests: $(VENV)
```

Comment:
:cry:  this means that if I want to use `make ci-unittests` I cannot manage my own env/virtualenv...
For my Python projects I use `virtualenvwrapper`, and my envs are not a subdirectory of the repo. I wish I could still use that :)

(the fact that we activate the env in every target already prevents me from managing my own env)